Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -428,10 +429,11 @@ void FullscreenManager::exitFullscreen(RefPtr<DeferredPromise>&& promise)
 
         // Notify the chrome of the new full screen element.
         if (mode == ExitMode::Resize)
-            page->chrome().client().exitFullScreenForElement(m_fullscreenElement.get());
+            page->chrome().client().exitFullScreenForElement(exitedFullscreenElement.get());
```

Comment:
Nit: It seems like `exitFullscreenForElement(..)` should take a RefPtr, since it does various operations inside that might affect the life cycle of the element.

Reply:
No, this is not how arguments in our safer C++ coding style work; the guidelines document explains this. @rniwa is the one who originally explained it to me.